src/utils/get-device-size.js

```diff
@@ -0,0 +1,23 @@
+import { readSync } from 'read-file-relative';
+import find from 'array-find';
+
+
+const DEVICE_DATABASE = JSON
+    .parse(readSync('../../data/devices.json'))
+    .map(device => {
+        return {
+            name:           device['Device Name'].toLowerCase().split(' ').join(''),
+            portraitWidth:  Number(device['Portrait Width']),
+            landscapeWidth: Number(device['Landscape Width'])
+        };
+    })
+    .filter(device => !isNaN(device.portraitWidth) && !isNaN(device.landscapeWidth));
```

Comment:
Do we need to require `update-database` from `build` task? Or use `publish-please` to update DB before publishing to `npm`?